sky/jobs/state.py

```diff
@@ -67,7 +67,9 @@ def create_table(cursor, conn):
         task_id INTEGER DEFAULT 0,
         task_name TEXT,
         specs TEXT,
-        local_log_file TEXT DEFAULT NULL)""")
+        local_log_file TEXT DEFAULT NULL,
+        pid INTEGER DEFAULT NULL,
+        dag_yaml_path TEXT)""")
```

Comment:
These two are needed for monitoring and starting (respectively) the job controller. They will only be set on the first task for any given job.